src/dash-table/DataTable.sanitize.ts

```diff
@@ -0,0 +1,58 @@
+import * as R from 'ramda';
```

Comment:
I would not normally do this but I've run into issues moving DataTable.js in the past -- normally I would do this:

DataTable
  |- index.js
  |- validate.ts
  |- sanitize.ts

It's been a long time and I certainly understand more about what's going on -- could give it another go.

Reply:
That would be fine... or leaving `DataTable.js` where it is and just giving these other ones other names without the extra dots - even `dataTableSanitize.ts` would make me more comfortable (I guess the leading capital also strikes me as off, since these don't define a class)